app/api/views/scores.py

```diff
@@ -50,12 +51,21 @@ def get(self, request):
             access perms require either:
             the user id in the API request matches the current user OR
             the current user has authorship permissions to the instance OR
-            the current user is a support user
+            the current user is a support user OR
+            the current user is an author or staff where request is launched from LTI
             """
+            is_author_or_staff = False
+            if LTILaunchService.is_lti_launch(request):
+                launch = LTILaunchService.get_launch_data(request)
+                is_author_or_staff = LTILaunchService.is_user_course_author(
+                    launch
+                ) or LTILaunchService.is_user_staff(launch)
+
             if (
                 request.user.id != user.id
                 and not instance.permissions.filter(user=request.user).exists()
                 and not PermService.is_superuser_or_elevated(request.user)
+                and not is_author_or_staff
```

Comment:
`is_author_or_staff` defaults to `False`, but it only has a chance of flipping to `True` when there's an LTI launch. That isn't always the case. We want to make sure this condition doesn't prevent non-LTI requests from succeeding.